src/modules/graph/graph.service.ts

```diff
@@ -9,7 +9,7 @@ export class GraphService {
   private readonly client: Client;
 
   constructor(graphClientService: GraphClientService) {
-    this.client = graphClientService.getClient();
+    this.client = graphClientService.client;
```

Comment:
The reason we moved away from the field in the first place was due to mocking in tests -- As we've developed a way to do so without requiring the getter, I moved it back

Reply:
There was no need for the getter (I think). It was originally a field. It's only consumed by graph.service. It was changed to a getter to get it easier to mock -- but since we've found a better way, I changed it back to a field